### PR TITLE
Enforce rustfmt in CI

### PR DESCRIPTION
**Pull request by Dirbaio**

### CODE REVIEW

**Review by crawford — APPROVED** [review]
Nice!

File: .github/workflows/rustfmt.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          protile: minimal
+          components: rustfmt
+      - name: Check fmt
+        run: cargo fmt -- --check
```
Comment (dlrobertson) [inline .github/workflows/rustfmt.yaml]: nit: newline :)

**Review by dlrobertson — APPROVED** [review]
:100:

File: .github/workflows/rustfmt.yaml (hunk 1 above)
Comment (Dirbaio) [reply .github/workflows/rustfmt.yaml]: fixed

**Pull request merged**